using_images/s2i_images/java.adoc

```diff
@@ -40,6 +40,9 @@ The RHEL 7 image is available through the Red Hat Registry:
 
 ----
 $ docker pull registry.redhat.io/redhat-openjdk-18/openjdk18-openshift
+$ docker pull registry.redhat.io/openjdk/openjdk-11-rhel7
+$ docker pull registry.redhat.io/ubi8/openjdk-8
+$ docker pull registry.redhat.io/ubi8/openjdk-11
```

Comment:
Regarding these image variants, are the bottom two variants for RHEL8?

Reply:
Yes that's right